server/fleet/hosts.go

```diff
@@ -216,17 +216,32 @@ func (h *Host) FleetPlatform() string {
 	return PlatformFromHost(h.Platform)
 }
 
+// HostLinuxOSs are the possible linux values for Host.Platform.
+// TODO(lucas): Osquery could also return `gentoo`, `posix`, `freebsd`.
+var HostLinuxOSs = []string{
+	"linux", "ubuntu", "debian", "rhel", "centos",
+}
+
+func isLinux(hostPlatform string) bool {
+	for _, linuxPlatform := range HostLinuxOSs {
+		if linuxPlatform == hostPlatform {
+			return true
+		}
+	}
+	return false
+}
+
 // PlatformFromHost converts the given host platform into
 // the generic platforms known by osquery
 // https://osquery.readthedocs.io/en/stable/deployment/configuration/
 // and supported by Fleet.
 //
 // Returns empty string if hostPlatform is unknnown.
 func PlatformFromHost(hostPlatform string) string {
-	switch hostPlatform {
-	case "ubuntu", "rhel", "debian":
+	switch {
+	case isLinux(hostPlatform):
```

Comment:
Do we have tests for this?

Reply:
You mean simple unit tests for `isLinux`?

Reply:
For `PlatformsForHost` and the logic that's being changed here. It's simple, but simple logic is where silly but important bugs happen :)

As in, I want to make sure that if we merge this now, we won't get a report saying "my linux hosts are not showing any data" or things like that because of a simple bug in this logic.

Reply:
+1, I'll add a simple unit test.

PS: `PlatformsForHost` is being indirectly used by `Host`, `ListPoliciesForHost` and `PolicyQueriesForHost`.